src/Frontend/Core/Js/jquery/jquery.frontend.js

```diff
@@ -337,41 +337,40 @@
  */
 (function($)
 {
-	$.fn.html5validation = function(options)
-	{
-		// define defaults
-		var $input = $(this),
-			errorMessage = '',
-			type = '',
-			defaults =
-			{
-				required: jsFrontend.locale.err('FieldIsRequired'),
-				email: 	  jsFrontend.locale.err('EmailIsInvalid'),
-				date: 	  jsFrontend.locale.err('DateIsInvalid'),
-				number:   jsFrontend.locale.err('NumberIsInvalid'),
-				value:    jsFrontend.locale.err('InvalidValue')
-			};
-
-		options = $.extend(defaults, options);
-
-		$input.on('invalid', function(e) {
-
-			if ($input.context.validity.valueMissing) {
-				errorMessage = options.required;
-			} else if (!$input.context.validity.valid) {
-				type = $input.context.type;
-				errorMessage = options.value;
-
-				if (options[type]) {
-					errorMessage = options[type];
-				}
-			}
-
-			e.target.setCustomValidity(errorMessage);
-
-			$input.on('input', function(e) {
-				e.target.setCustomValidity('');
-			});
-		});
-	};
+    $.fn.html5validation = function(options)
+    {
+        // define defaults
+        var $input = $(this),
+            errorMessage = '',
+            type = '',
+            defaults = {
+                required: jsFrontend.locale.err('FieldIsRequired'),
+                email: 	  jsFrontend.locale.err('EmailIsInvalid'),
+                date: 	  jsFrontend.locale.err('DateIsInvalid'),
+                number:   jsFrontend.locale.err('NumberIsInvalid'),
+                value:    jsFrontend.locale.err('InvalidValue')
+            };
+
+        options = $.extend(defaults, options);
+
+        $input.on('invalid', function(e) {
+
+            if ($input.context.validity.valueMissing) {
+                errorMessage = options.required;
+            } else if (!$input.context.validity.valid) {
+                type = $input.context.type;
+                errorMessage = options.value;
+
+                if (options[type]) {
+                    errorMessage = options[type];
+                }
+            }
+
+            e.target.setCustomValidity(errorMessage);
+
+            $input.on('input change', function(e) {
```

Comment:
it is called, it is apart from the indentation fix the actual place where I changed something because it didn't work properly